server/server.js

```diff
@@ -41,6 +43,23 @@ if (process.env.ENV === "PRODUCTION") {
   });
 }
 
+app.get("/authenticate/user-posts", (req, res) => {
+  const { ghUsername, githubId, avatarUrl } = req.query;
+  User.find({ username: ghUsername })
```

Comment:
```suggestion
  const result = await User.findOne({ username: ghUsername });
```

can we change this to `await` too or do we need the `then`?

Reply:
any code you await also needs to be wrapped in a `try` `catch` in case the promise fails I think